test/parallel/test-http2-binding.js

```diff
@@ -13,7 +13,7 @@ assert(http2.SessionShutdownWrap);
 assert.strictEqual(typeof http2.Http2Session, 'function');
 assert.strictEqual(typeof http2.SessionShutdownWrap, 'function');
 
-const settings = http2.getDefaultSettings({});
+const settings = require('http2').getDefaultSettings();
```

Comment:
why is require used here, I believe the module cache delivers ithe module anyway from the cache?